libs/community/tests/integration_tests/vectorstores/test_falkordbvector.py

```diff
@@ -0,0 +1,596 @@
+"""Test FalkorDB functionality"""
+
+import os
+from math import isclose
+from typing import Any, Dict, List, cast 
+
+
+from langchain_community.graphs import FalkorDBGraph
+from langchain_core.documents import Document
+from langchain_community.vectorstores.falkordb_vector import (
+    FalkorDBVector,
+    SearchType,
+    _get_search_index_query,
+    process_data
+)
+from langchain_community.vectorstores.utils import DistanceStrategy
+from tests.integration_tests.vectorstores.fake_embeddings import (
+    AngularTwoDimensionalEmbeddings,
+    FakeEmbeddings,
+)
+
+import enum
+
+class SearchType(str, enum.Enum):
+    """Enumerator of the Distance strategies."""
+
+    VECTOR = "vector"
+    HYBRID = "hybrid"
+
+
+
+#NOTE: This test is conducted on a local FalkorDB but can be carried 
+# out on a Cloud FalkorDB too.
+
+host = 'localhost'
+port = 6379
```

Comment:
maybe use env vars with default value?

Reply:
Just to be clear, you want me to change those lines of code to
```
host = os.getenv("FALKORDB_HOST")
port = os.getenv("FALKORDB_PORT")
```